test/cpp/nannew.cpp

```diff
@@ -342,9 +376,13 @@ NAN_METHOD(testRegression242) {
   NanScope();
   NanTap t(args[0]);
 
-  // This line needs to *compile*. Not much to test at runtime.
+  // These lines must *compile*. Not much to test at runtime.
   Local<FunctionTemplate> ft = NanNew<FunctionTemplate>(overloaded);
   (void)ft;  // not unused
+#if (NODE_MODULE_VERSION >= 12)
+  Local<Function> f = NanNew<Function>(overloaded);
+  (void)f;  // not unused
+#endif
```

Comment:
Nope. Fixed. (Sorry, about the spam).

On 22.01.2015, at 14:16, Benjamin Byholm <notifications@github.com> wrote:

> In test/cpp/nannew.cpp:
> 
> >    Local<FunctionTemplate> ft = NanNew<FunctionTemplate>(overloaded);
> >    (void)ft;  // not unused
> > +#if (NODE_MODULE_VERSION >= 12)
> > +  Local<Function> f = NanNew<Function>(overloaded);
> > +  (void)f;  // not unused
> > +#endif
> 
> Is this conditioning necessary any more?
> 
> —
> Reply to this email directly or view it on GitHub.
>